redis/content.md

```diff
@@ -62,10 +62,11 @@ The mapped directory should be writable, as depending on the configuration and m
 
 This variant is *not* a 32bit image (and will not run on 32bit hardware), but includes Redis compiled as a 32bit binary, especially for users who need the decreased memory requirements associated with that. See ["Using 32 bit instances"](http://redis.io/topics/memory-optimization#using-32-bit-instances) in the Redis documentation for more information.
 
-# Redis Modules
+# Redis Stack and modules
 
-You can find the list of modules for Redis on [redis.io](https://redis.io/modules) or on [redismodules.com](http://redismodules.com). A few of the standard modules can be found here:
+Modules extend Redis with extra capabilities. Redis Stack already comes with the capabilities (search and query, JSON support, and more) that are developed by Redis (the company). The Docker images are available here:
 
--	[RediSearch](https://hub.docker.com/r/redislabs/redisearch/): Search and Query with Indexing on Redis
--	[ReJSON](https://hub.docker.com/r/redislabs/rejson/): Extended JSON processing for Redis
--	[ReBloom](https://hub.docker.com/r/redislabs/rebloom/): Bloom Filters data type for membership/existence search on Redis
+-	[Redis Stack](https://hub.docker.com/r/redis/redis-stack): Redis plus extra capabilities and Redis Insight (a GUI for Redis)
+-	[Redis Stack Server](https://hub.docker.com/r/redis/redis-stack-server): Redis plus extra capabilities
```

Comment:
Have the images under the `redislabs` namespace been deprecated?

Reply:
No, the other container images are still valid. However, we prefer to bundle and message our modules together in what's called Redis Stack. The search and query module, for instance, relies on the JSON module for indexing JSON documents. So we think it makes sense to put the emphasis on Redis Stack and to talk about its combined capabilities rather than the individual modules.

Reply:
Hi David,

I fixed those two minor things and commented in regards to the Stack image. Please let me know if we are good to merge the changes now.

Regards,
David